src/main/resources/static/css/academicCourseSectionForTeachersStyle.css

```diff
@@ -635,6 +635,7 @@ form {
     border: 1px solid #01A7F0;
 }
 .sidebar2 {
+    margin-right: 25px;
     margin-top: 50px;
```

Comment:
The sidebar2 actually needs to be in a seperate div and margin-right: 25px along with margin-top: 50px is not an appropriate solution, though currently it might be accepted.